hack/lib/catalogsource.bash

```diff
@@ -12,13 +12,18 @@ function ensure_catalogsource_installed {
 function install_catalogsource {
   logger.info "Installing CatalogSource"
 
-  # Determine if we're running locally or in CI.
-  if [ -z "$OPENSHIFT_BUILD_NAMESPACE" ]; then
-    export IMAGE_KNATIVE_SERVING_OPERATOR="${DOCKER_REPO_OVERRIDE}/knative-serving-operator"
-    export IMAGE_KNATIVE_OPENSHIFT_INGRESS="${DOCKER_REPO_OVERRIDE}/knative-openshift-ingress"
-  else
+  # Use current build images in CI.
+  if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
     export IMAGE_KNATIVE_SERVING_OPERATOR="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:knative-serving-operator"
     export IMAGE_KNATIVE_OPENSHIFT_INGRESS="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:knative-openshift-ingress"
+  # Use DOCKER_REPO_OVERRIDE for developing locally
+  elif [ -n "$DOCKER_REPO_OVERRIDE" ]; then
+    export IMAGE_KNATIVE_SERVING_OPERATOR="${DOCKER_REPO_OVERRIDE}/knative-serving-operator"
+    export IMAGE_KNATIVE_OPENSHIFT_INGRESS="${DOCKER_REPO_OVERRIDE}/knative-openshift-ingress"
+  # Use CI_PROMOTION_NAME to use current promoted images from CI
```

Comment:
Wondering if we should hardcode this to the actual version maybe? With the scripts in this branch, only the 1.3.0 promotion is supposed to work, right? Maybe we can even roll without the new variable then.

Reply:
I think the only problem is that you will need to update this script with every new branch.

Reply:
We kinda need to do that anyway, we have references to v0.10 and stuff